odt-opt-in/ODT Opt-in JSON Contract.md

```diff
@@ -173,16 +173,25 @@ UX returns updated state of each share. Share IDs listed are to be enrolled into
 
 ## Error Codes
 
-| Error Code | Logging Error Message                                     |
-| ---------- | --------------------------------------------------------- |
-| 501        | Config file read error: _[read error]_                    |
-| 502        | Config file validation error: _[validation error detail]_ |
-| 503        | No eligible shares                                        |
-| 504        | Ineligible account type                                   |
-| 505        | Account warning found                                     |
-| 506        | Error attempting to update share tracking record          |
-| 507        | Error updating source code and auth & fee fields          |
-| 508        | Error updating share overdraft tolerance amount           |
+*See the Modifier section for additional details.
+ The Modifier is appended to the main Logging Error Message.
+
+| Error Code | Logging Error Message                                                                 | Modifier                                                                             | Additional Notes As Needed                    |
+| ---------- | ------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------- |-----------------------------------------------|
+| 500        | Program running in memo mode |||
+| 501        | Error reading from config file | :Error Opening Letterfile [configuration file name]: [system generated letter file read error message] ||
+|            || :Error Reading Letterfile [configuration file name]: [system generated letter file read error message] ||
+| 502        | Config file validation error | :Invalid tracking type||
+|            || :Invalid source code for SCT in CFG | In the CFG for SCT (source code 1 value if opt-in = true) there is an invalid value. Remove the value from the CFG not listed as valid in CFG comments. |
```

Comment:
I think the last line of these 4 errors is a bit confusing. Can we say something like "See CFG comments for valid values."?

Reply:
Thank you Justin!

That sounds better.
I have made those changes in commit 2.